src/vmm/src/devices/virtio/balloon/device.rs

```diff
@@ -762,6 +762,12 @@ pub(crate) mod tests {
         // Make sure nothing got written.
         balloon.read_config(0, &mut actual_config_space);
         assert_eq!(actual_config_space, expected_config_space);
+
+        // Large offset that may cause an overflow.
+        balloon.write_config(0xffff_ffff_ffff_ffff, &new_config_space);
```

Comment:
nit: maybe use `u64::MAX` instead of hard-coding the value?

Reply:
indeed, done, thanks